tests/python/quantization/test_quantization.py

```diff
@@ -747,6 +747,7 @@ def get_fp32_sym_with_multiple_outputs(length=1):
     return sym
 
 @with_seed()
+@unittest.skip('skip for MKL-DNN 1.0 integration: https://github.com/apache/incubator-mxnet/projects/16')
```

Comment:
@ZhennanQin could we switch quantization tests now? I see several INT8 PRs are already merged.

Reply:
rebase and enable now, disable due to subgrap fc PR dependence